fe/fe-core/src/main/java/com/starrocks/clone/TabletScheduler.java

```diff
@@ -556,6 +556,9 @@ private void scheduleTablet(TabletSchedCtx tabletCtx, AgentBatchTask batchTask)
             if (tbl == null) {
                 throw new SchedException(Status.UNRECOVERABLE, "tbl does not exist");
             }
+            if (tbl.isLakeTable()) {
+                throw new SchedException(Status.UNRECOVERABLE, "tablet is managed by StarOS");
```

Comment:
suggest change message a little bit. e.g. `tablet is not managed by fe` or `tablet is managed externally.`